README.md

```diff
@@ -9,51 +9,35 @@ with a fully functional system straight from your database schema. If
-- [Schemas](#GettingStartedHatchify-Schemas)
-  - [Model
-    Relationships](#GettingStartedHatchify-ModelRelationships)
-- [Backend - The Hatchify
-  Middleware](#GettingStartedHatchify-Backend-TheHatchifyMiddleware)
-  - [Using the server
-    endpoints](#GettingStartedHatchify-Usingtheserverendpoints)
-    - [Creating a
-      resource](#GettingStartedHatchify-Creatingaresource)
-    - [Listing
-      resources](#GettingStartedHatchify-Listingresources)
-    - [Fetching a single
-      resource](#GettingStartedHatchify-Fetchingasingleresource)
-    - [Updating a
-      resource](#GettingStartedHatchify-Updatingaresource)
-    - [Deleting a
-      resource](#GettingStartedHatchify-Deletingaresource)
-  - [Seeding data](#GettingStartedHatchify-Seedingdata)
-- [Frontend with React &
-  MUI](#GettingStartedHatchify-FrontendwithReact&MUI)
-  - [Rendering a List](#GettingStartedHatchify-RenderingaList)
-- [Next Steps](#GettingStartedHatchify-nextSteps)
-  - [Rendering an empty list](#GettingStartedHatchify-renderingAnEmptyList)
-- [Need help or have
-  questions?](#GettingStartedHatchify-Needhelporhavequestions?)
+Unlike code generation tools—which allow you to write your schema and then generate your code, but once you start making customizations you can't re-run the generator without losing your customizations—HatchifyJS enables you to make changes to your database schema and customize app behavior **independently**. This is because HatchifyJS is **not** code generation—it's a system of modular and hierarchical libraries that can be consumed piecemeal to use as much or as little of HatchifyJS abilities as you require.
+
+- [HatchifyJS](#hatchifyjs)
+- [Project Setup](#project-setup)
+- [Schemas](#schemas)
+  - [Model Relationships](#model-relationships)
+- [Backend - The Hatchify Middleware](#backend---the-hatchify-middleware)
+  - [Using the server endpoints](#using-the-server-endpoints)
+    - [Creating a resource](#creating-a-resource)
+    - [Listing resources](#listing-resources)
+    - [Fetching a single resource](#fetching-a-single-resource)
+    - [Updating a resource](#updating-a-resource)
+    - [Deleting a resource](#deleting-a-resource)
+  - [Seeding data](#seeding-data)
+- [Frontend with React & MUI](#frontend-with-react--mui)
+  - [Rendering a List](#rendering-a-list)
+- [Next Steps](#next-steps)
+  - [Rendering an empty list](#rendering-an-empty-list)
+- [Need help or have questions?](#need-help-or-have-questions)
 
 # Project Setup
 
-In this guide, we will be setting up one project containing a HatchifyJS
-frontend and backend. Our frontend will use React & MUI and our backend
+In this guide, we will be setting up a project containing a HatchifyJS
+frontend and backend. Our frontend will use React and MUI and our backend
 will be using Koa. We will be setting up our project using Vite so that
 it handles all the React configurations for us.
 
+> Note: The ✏️ icon indicates when to follow along!
```

Comment:
The ✏️ icon convention is common at Bitovi, but I've never seen it anywhere else